lib/aganakti/log_subscriber.rb

```diff
@@ -0,0 +1,78 @@
+# frozen_string_literal: true
+
+module Aganakti
+  ##
+  # A log subscriber using the +ActiveSupport::LogSubscriber+ interface. It only logs anything if the logger
+  # has debug level enabled.
+  #
+  # @see ActiveSupport::LogSubscriber
+  # @see https://api.rubyonrails.org/classes/ActiveSupport/LogSubscriber.html ActiveSupport::LogSubscriber in the Rails docs
+  class LogSubscriber < ActiveSupport::LogSubscriber
+    ##
+    # Called by ActiveSupport notifications when there's a SQL query that executes
```

Comment:
Missing doc for param and return?

Reply:
It's a void, but I added the doc for `@param`